capten/config-worker/internal/crossplane/config_provider_update.go

```diff
@@ -0,0 +1,32 @@
+package crossplane
+
+import (
+	"context"
+
+	"github.com/kube-tarian/kad/capten/model"
+	agentmodel "github.com/kube-tarian/kad/capten/model"
+	"github.com/pkg/errors"
+)
+
+const (
+	CrossPlaneResource  = "crossplane"
+	CrossplaneNamespace = "crossplane-system"
+)
+
+func (cp *CrossPlaneApp) configureConfigProviderUpdate(ctx context.Context, req *model.CrossplaneClusterUpdate) (status string, err error) {
+	logger.Infof("configuring config provider %s update", req.ManagedClusterName)
+
+	err = cp.helper.SyncArgoCDApp(ctx, CrossPlaneResource, CrossPlaneResource)
+	if err != nil {
+		return string(agentmodel.WorkFlowStatusFailed), errors.WithMessage(err, "failed to sync config providers")
+	}
```

Comment:
it is require to sync provider k8s-config, like aws-k8s-package.
implementation not correct, it just duplicate the other code